src/__tests__/components/CreateArticle.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { CreateArticle } from '../../components/Articles/CreateArticle.jsx';
+
+// mock for mutations observer window object
+global.MutationObserver = class {
+  constructor(_callback) {}
+
+  disconnect = () => {}
+
+  observe(_element, _initObject) {}
+};
+
+describe('Renders CreateArticleComponent', () => {
+  const props = {
+    article: {
+      message: 'test',
+    },
+    title: '<h1>Write my name</h1>',
+    body: '<p>What did we have for lunch</p>',
+    error: '',
+    publishArticle: jest.fn(),
+  };
+
+  const wrapper = shallow(
+    <CreateArticle {...props} />,
+  );
+  it('should render create component', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('should render a mainDiv class', () => {
+    expect(wrapper.find('.mainDiv').length).toBe(1);
+  });
+  it('should render a publishBtn class', () => {
+    expect(wrapper.find('.publishBtn').length).toBe(1);
```

Comment:
as a `TTL` forexample i need to show me that since you are testing components and using `find` to find button elements, i want you to go a head and simulate the `click` event to test if your buttons respond to `events`.